views/css/avarda.css

```
@@ -1,102 +1,137 @@
 #avarda-checkout {
-    margin-bottom: 3em;
+  margin-bottom: 3em;
 }
 
-/*
-body#module-avardapayments-checkout #header .header-nav {
-    max-height: none;
-    padding: 5rem 0 .9375rem;
-    border: none;
-    margin-bottom: 0;
-    box-shadow: 2px 2px 8px 0 rgba(0,0,0,.2);
+#module-avardapayments-opcheckout #header .header-nav {
```

```suggestion
#module-avardapayments-opcheckout #header .header-nav, 
#module-avardapayments-checkout #header .header-nav {
```
to get this to regular flow also